### PR TITLE
docs: pull in loopback 4 todo example tutorial docs

### PR DESCRIPTION
**Pull request by virkt25**
- Add example-getting-started docs to script to pull those in as a loopback 4 readme
- update travis config
- fix package.json script

connected to https://github.com/strongloop/loopback-next/issues/729

### CODE REVIEW

**Review by raymondfeng — APPROVED** [review]

**Comment by shimks** [comment]
Should this PR also bring in the readmes/tutorial files the script is supposed to bring in? Also, is the script going to work similarly to the one that brings over stuff in `@loopback/docs`?

**Review by b-admike — APPROVED** [review]

**Comment by virkt25** [comment]
@shimks Travis will pull in the missing files as the script to pull in these files is run by Travis (same as `@loopback/docs`.

**Review by shimks — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
   "name": "loopback.io-workflow-scripts",
   "version": "1.0.0",
-  "description": "This file: Node-dependant workflow scripts for the jekyll-based site",
+  "description":
+    "This file: Node-dependant workflow scripts for the jekyll-based site",
   "scripts": {
     "start": "bundle exec jekyll serve --incremental",
-    "fetch-readmes": "get-readmes --repos=./_data --out=./_includes/readmes",
+    "fetch-readmes":
+      "./update-readmes.sh --repos=./_data --out=./_includes/readmes && ./update-v4-readmes.sh",
```
Comment (bajtos) [inline package.json:9]: Please keep key/value pairs on a single line, that's what `npm` will do the next time we run any `npm` command modifying `package.json`. The same comment applies to `description` too.

Comment (virkt25) [reply package.json:9]: My apologies. That was auto-formatted on save by editor (Prettier extension) as I didn't have config to disable it from `.json` files. I didn't realize this had happened. 

I was going to make a PR to fix it but turns out the Travis build already did so in the last build https://github.com/strongloop/loopback.io/commit/5871ba370f189c3bd15e276396a1146da8c580b6

**Pull request merged**